web/libs/editor/src/components/App/App.jsx

```diff
@@ -104,7 +104,7 @@ class App extends Component {
         <Result status="success" title={getEnv(this.props.store).messages.NO_NEXT_TASK} />
         <Block name="sub__result">You have completed all tasks in the queue!</Block>
         {store.taskHistory.length > 0 && (
-          <Button onClick={(e) => store.prevTask(e, true)} look="outlined" style={{ margin: "16px 0" }}>
+          <Button onClick={(e) => store.prevTask(e, true)} variant="neutral" style={{ margin: "16px 0" }}>
```

Comment:
there is likely a tailwind class for this - probably something like `mx-0 my-[16]`